katcp/server.py

```diff
@@ -2241,10 +2249,22 @@ def request_sensor_sampling(self, req, msg):
             ?sensor-sampling cpu.power.on period 500
             !sensor-sampling ok cpu.power.on period 500
 
+            if BULK_SET_SENSOR_SAMPLING is enabled then:
+
+            ?sensor-sampling cpu.power.on fan.speed
+            !sensor-sampling fail Cannot\_query\_multiple\_sensors
+
+            ?sensor-sampling cpu.power.on fan.speed period 500
+            !sensor-sampling ok cpu.power.on fan.speed period 500
```

Comment:
Would have been nicer to have the name or list of names at the end - but unfortunately legacy dictates otherwise...